cartaya/4/task4.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style4.css">
+    <title>Task#4</title>
+</head>
+<body>
+    <section class="section-left">
+        <h1 class="title">Directory Profile Name</h1>
+        <div class="profile-img"><img src="perfil.png" alt="profile picture" title="profile" height="152px" width="150.5"></div>
+        <div class="profile-databox">
+            <h1 class="h1-profile-name">Profile Name</h1>
+            <div class="name profile-data">
+                <div class="profile-data-img"><img src="1.png" alt="name" width="25px" height="15px"></div>

```

Comment:
Don't use the width and height attribute intead use CSS styles. We need to follow the separation of concerns principle, the styles always in CSS.